server/index.js

```diff
@@ -1,10 +1,14 @@
 const express = require('express')
+const path = require('path')
 const router = require('./router')
 
 const app = express()
+app.use(express.static(path.join(__dirname, '..', 'client', 'dist')))
```

Comment:
Possible action 2:
Change `app.use(express.static(path.join(__dirname, '..', 'client', 'dist')))` to `app.use(express.static(path.join(__dirname, '..', 'client', 'src')))`

Reply:
Thanks. There is a build step that occurs. See previous comment.